package.json

```diff
@@ -32,7 +32,8 @@
     "react-icons": "^5.3.0",
     "react-router-dom": "^6.21.3",
     "web-vitals": "^3.5.2",
-    "zod": "^3.23.8"
+    "zod": "^3.23.8",
+    "zustand": "^5.0.0-rc.2"
```

Comment:
Maybe it is worth mentioning in the PR description that zustand is taken into use.